transcom/mymove
# B-20472 customer cancel move

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A980088&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

Adds the ability for customers to cancel moves before submitting.

### How to test

1. Login to the customer app
2. Create new move
3. Add orders information
4. Click 'Cancel Move'
5. Confirm cancel move
6. Move is canceled.
